test/TesterInternal/OrleansRuntime/Streams/PooledQueueCacheTests.cs

```diff
@@ -236,15 +236,16 @@ private int RunGoldenPath(PooledQueueCache<TestQueueMessage, TestCachedMessage>
 
             // now add messages into cache newer than cursor
             // Adding enough to fill the pool
-            for (int i = 0; i < MessagesPerBuffer * PooledBufferCount; i++)
-            {
-                cache.Add(new TestQueueMessage
+            List<TestQueueMessage> messages = Enumerable.Range(0, MessagesPerBuffer * PooledBufferCount)
+                .Select(i => new TestQueueMessage
                 {
                     StreamGuid = i % 2 == 0 ? stream1.Guid : stream2.Guid,
                     StreamNamespace = TestStreamNamespace,
-                    SequenceNumber = sequenceNumber++,
-                }, DateTime.UtcNow);
-            }
+                    SequenceNumber = sequenceNumber + i
```

Comment:
why it this `SequenceNumber = sequenceNumber + i `, instead of the original logic of  `sequenceNumber++ `? seems like you changed the logic here. And btw this test is failing.

Reply:
I was trying to avoid modifying state in an enumeration, can be hard to maintain.
Introduced cut+paste error, reusing 'messages' rather than the new 'moremessages'